### ISSUE TITLE
flutter build ios throws rsync: link stat "...App.framework" failed: No such file or directory

### ISSUE THREAD

**Issue by arunk** [flutter/flutter#98011]
## Steps to Reproduce

1. Create a project with `flutter create newproject`
2. Change directory to newproject
3. Execute `flutter run` on the sample app. It runs on a device physical or simulator.
4. Execute `flutter build ios` on the sample app.
5. Get error 'Command PhaseScriptExecution failed with a nonzero exit code'
6. Tick the 'For install builds only' in 'Run Scripts' in the Xcode as mentioned [here](https://stackoverflow.com/questions/57919267/command-phasescriptexecution-failed-with-a-nonzero-exit-code-while-trying-to-add)
7. Execute `flutter build ios` again.
8. Get the above error mentioned in the title of this issue.

**Expected results:** 
For the build to succeed and to have an IOS application.

**Actual results:** 
```
    $ flutter build ios  
    Building com.myndintranquility.iosapp for device (ios-release)...
    Automatically signing iOS for device deployment using specified development team in Xcode project: 6RR32KRHR5
    Running Xcode build...                                                  
    Xcode build done.                                           12.9s
    Failed to build iOS app
    Error output from Xcode build:
    ↳
        ** BUILD FAILED **


    Xcode's output:
    ↳
        Writing result bundle at path:
    	    /var/folders/mh/_vvkkg514w96pb346l96plj80000gn/T/flutter_tools.b61GWA/flutter_ios_build_temp_dirufev78/temporary_xc
    	result_bundle

    rsync: link_stat "/Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework" failed: No such
    file or directory (2)
    rsync error: some files could not be transferred (code 23) at
    /System/Volumes/Data/SWE/macOS/BuildRoots/533514bb11/Library/Caches/com.apple.xbs/Sources/rsync/rsync-55/rsync/main.
    c(996) [sender=2.6.9]
    rsync: link_stat "/Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework" failed: No such
    file or directory (2)
    rsync error: some files could not be transferred (code 23) at
    /System/Volumes/Data/SWE/macOS/BuildRoots/533514bb11/Library/Caches/com.apple.xbs/Sources/rsync/rsync-55/rsync/main.
    c(996) [sender=2.6.9]

    Unhandled exception:
    Exception: Command "rsync -av --delete --filter - .DS_Store
    /Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework
    /Users/new/Development/myndintranquility/build/ios/Release-iphoneos/Runner.app/Frameworks" exited with code 23
    #0      Context.runSync (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:105:7)
    #1      Context.embedFlutterFrameworks
    (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:190:5)
    #2      Context.run (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:61:9)
    #3      main (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:17:5)
    #4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
    #5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    /Users/new/Development/myndintranquility/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target
    'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.2.99.
    (in target 'FMDB' from project 'Pods')

    Result bundle written to path:
    	/var/folders/mh/_vvkkg514w96pb346l96plj80000gn/T/flutter_tools.b61GWA/flutter_ios_build_temp_dirufev78/temporary_xc
    	result_bundle


Encountered error while building for device.
```
I came across this issue #63150 which refers to the same error but the fix of upgrading the flutter_svg doesn't work because I'm not using that library.

<details>
<summary>Code sample</summary>

```dart
```

</details>

<details>
  <summary>Logs</summary>

```
$ flutter build ios
Building com.myndintranquility.iosapp for device (ios-release)...
Automatically signing iOS for device deployment using specified development team in Xcode project: 6RR32KRHR5
Running Xcode build...                                                  
Xcode build done.                                           12.9s
Failed to build iOS app
Error output from Xcode build:
↳
    ** BUILD FAILED **


Xcode's output:
↳
    Writing result bundle at path:
    	/var/folders/mh/_vvkkg514w96pb346l96plj80000gn/T/flutter_tools.b61GWA/flutter_ios_build_temp_dirufev78/temporary_xc
    	result_bundle

    rsync: link_stat "/Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework" failed: No such
    file or directory (2)
    rsync error: some files could not be transferred (code 23) at
    /System/Volumes/Data/SWE/macOS/BuildRoots/533514bb11/Library/Caches/com.apple.xbs/Sources/rsync/rsync-55/rsync/main.
    c(996) [sender=2.6.9]
    rsync: link_stat "/Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework" failed: No such
    file or directory (2)
    rsync error: some files could not be transferred (code 23) at
    /System/Volumes/Data/SWE/macOS/BuildRoots/533514bb11/Library/Caches/com.apple.xbs/Sources/rsync/rsync-55/rsync/main.
    c(996) [sender=2.6.9]

    Unhandled exception:
    Exception: Command "rsync -av --delete --filter - .DS_Store
    /Users/new/Development/myndintranquility/build/ios/Release-iphoneos/App.framework
    /Users/new/Development/myndintranquility/build/ios/Release-iphoneos/Runner.app/Frameworks" exited with code 23
    #0      Context.runSync (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:105:7)
    #1      Context.embedFlutterFrameworks
    (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:190:5)
    #2      Context.run (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:61:9)
    #3      main (file:///Users/new/Development/flutter/packages/flutter_tools/bin/xcode_backend.dart:17:5)
    #4      _delayEntrypointInvocation.<anonymous closure> (dart:isolate-patch/isolate_patch.dart:295:32)
    #5      _RawReceivePortImpl._handleMessage (dart:isolate-patch/isolate_patch.dart:192:12)
    Command PhaseScriptExecution failed with a nonzero exit code
    note: Using new build system
    note: Planning
    note: Build preparation complete
    note: Building targets in dependency order
    /Users/new/Development/myndintranquility/ios/Pods/Pods.xcodeproj: warning: The iOS deployment target
    'IPHONEOS_DEPLOYMENT_TARGET' is set to 8.0, but the range of supported deployment target versions is 9.0 to 15.2.99.
    (in target 'FMDB' from project 'Pods')

    Result bundle written to path:
    	/var/folders/mh/_vvkkg514w96pb346l96plj80000gn/T/flutter_tools.b61GWA/flutter_ios_build_temp_dirufev78/temporary_xc
    	result_bundle


Encountered error while building for device.

```

```
info • Unused import: 'package:flutter_bloc/flutter_bloc.dart' •
          lib/api_connection/api_connection.dart:5:8 • unused_import
   info • Unused import:
          'package:myndintranquility/bloc/authentication_bloc.dart' •
          lib/api_connection/api_connection.dart:6:8 • unused_import
   info • Prefer const over final for declarations •
          lib/api_connection/api_connection.dart:17:1 •
          prefer_const_declarations
   info • Prefer const over final for declarations •
          lib/api_connection/api_connection.dart:19:1 •
          prefer_const_declarations
   info • Prefer const over final for declarations •
          lib/api_connection/api_connection.dart:21:1 •
          prefer_const_declarations
   info • The declaration '_registerURL' isn't referenced •
          lib/api_connection/api_connection.dart:24:7 • unused_element
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:27:3 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:38:5 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:60:5 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:61:5 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:86:5 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:87:5 • avoid_print
   info • Avoid `print` calls in production code •
          lib/api_connection/api_connection.dart:118:3 • avoid_print
   info • 'showSnackBar' is deprecated and shouldn't be used. Use
          ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/api_connection/api_connection.dart:137:28 •
          deprecated_member_use
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:138:26 •
          prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:141:18 •
          prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:143:19 •
          prefer_const_constructors
   info • 'showSnackBar' is deprecated and shouldn't be used. Use
          ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/api_connection/api_connection.dart:148:26 •
          deprecated_member_use
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:149:24 •
          prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:152:16 •
          prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/api_connection/api_connection.dart:154:17 •
          prefer_const_constructors
   info • Unused import: 'package:meta/meta.dart' •
          lib/bloc/authentication_bloc.dart:4:8 • unused_import
   info • Use key in widget constructors • lib/common/loading_indicator.dart:3:7
          • use_key_in_widget_constructors
   info • Prefer const with constant constructors •
          lib/common/loading_indicator.dart:5:41 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/common/loading_indicator.dart:6:16 • prefer_const_constructors
   info • Prefer typing uninitialized variables and fields •
          lib/dao/user_dao.dart:10:9 • prefer_typing_uninitialized_variables
   info • Avoid `print` calls in production code • lib/dao/user_dao.dart:14:7 •
          avoid_print
   info • Avoid `print` calls in production code • lib/dao/user_dao.dart:19:9 •
          avoid_print
   info • Use isNotEmpty instead of length • lib/dao/user_dao.dart:37:11 •
          prefer_is_empty
   info • Use isNotEmpty instead of length • lib/dao/user_dao.dart:51:11 •
          prefer_is_empty
   info • Prefer const over final for declarations •
          lib/database/user_database.dart:8:1 • prefer_const_declarations
   info • The import of 'package:flutter/widgets.dart' is unnecessary because
          all of the used elements are also provided by the import of
          'package:flutter/material.dart' • lib/home/categories_page.dart:2:8 •
          unnecessary_import
   info • Unused import: 'package:myndintranquility/model/api_model.dart' •
          lib/home/categories_page.dart:6:8 • unused_import
   info • Prefer typing uninitialized variables and fields •
          lib/home/categories_page.dart:20:9 •
          prefer_typing_uninitialized_variables
   info • The exception variable 'e' isn't used, so the 'catch' clause can be
          removed • lib/home/categories_page.dart:24:28 • unused_catch_clause
   info • Avoid `print` calls in production code •
          lib/home/categories_page.dart:25:9 • avoid_print
   info • Prefer const with constant constructors •
          lib/home/categories_page.dart:37:16 • prefer_const_constructors
   info • Avoid unnecessary containers • lib/home/categories_page.dart:58:13 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/home/categories_page.dart:79:33 • prefer_const_constructors
   info • Use key in widget constructors • lib/home/home_page.dart:5:7 •
          use_key_in_widget_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:10:16 • prefer_const_constructors
   info • Avoid unnecessary containers • lib/home/home_page.dart:12:13 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:17:13 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:17:30 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:18:20 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:20:22 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:25:24 • prefer_const_constructors
   info • SizedBox for whitespace • lib/home/home_page.dart:26:22 •
          sized_box_for_whitespace
   info • 'RaisedButton' is deprecated and shouldn't be used. Use ElevatedButton
          instead. See the migration guide in
          flutter.dev/go/material-button-migration-guide). This feature was
          deprecated after v1.26.0-18.0.pre. • lib/home/home_page.dart:29:24 •
          deprecated_member_use
   info • 'RaisedButton.RaisedButton' is deprecated and shouldn't be used. Use
          ElevatedButton instead. See the migration guide in
          flutter.dev/go/material-button-migration-guide). This feature was
          deprecated after v1.26.0-18.0.pre. • lib/home/home_page.dart:29:24 •
          deprecated_member_use
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:30:26 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:32:28 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:40:26 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/home_page.dart:41:27 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/questions_page.dart:52:16 • prefer_const_constructors
   info • Avoid unnecessary containers • lib/home/questions_page.dart:54:13 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/home/questions_page.dart:106:38 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/questions_page.dart:114:38 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/questions_page.dart:218:46 • prefer_const_constructors
   info • Use key in widget constructors • lib/home/registration_page.dart:4:7 •
          use_key_in_widget_constructors
   info • The value of the field '_usernameController' isn't used •
          lib/home/registration_page.dart:10:9 • unused_field
   info • The value of the field '_emailController' isn't used •
          lib/home/registration_page.dart:11:9 • unused_field
   info • The value of the field '_bioController' isn't used •
          lib/home/registration_page.dart:12:9 • unused_field
   info • The value of the field '_passwordController' isn't used •
          lib/home/registration_page.dart:13:9 • unused_field
   info • The value of the field '_confirmPasswordController' isn't used •
          lib/home/registration_page.dart:14:9 • unused_field
   info • The value of the field '_firstNameController' isn't used •
          lib/home/registration_page.dart:15:9 • unused_field
   info • The value of the field '_lastNameController' isn't used •
          lib/home/registration_page.dart:16:9 • unused_field
   info • Prefer declaring const constructors on `@immutable` classes •
          lib/home/resource_view.dart:8:3 •
          prefer_const_constructors_in_immutables
   info • Prefer const with constant constructors •
          lib/home/resource_view.dart:18:18 • prefer_const_constructors
   info • The import of 'package:flutter/rendering.dart' is unnecessary because
          all of the used elements are also provided by the import of
          'package:flutter/material.dart' • lib/home/surveydetail_page.dart:2:8
          • unnecessary_import
   info • Unnecessary string interpolation •
          lib/home/surveydetail_page.dart:15:21 •
          unnecessary_string_interpolations
   info • Avoid unnecessary containers • lib/home/surveydetail_page.dart:17:13 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:21:15 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:22:22 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:25:15 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:25:32 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:27:15 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:27:32 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:34:26 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:35:28 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:41:25 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/home/surveydetail_page.dart:42:33 • prefer_const_constructors
   info • Prefer typing uninitialized variables and fields •
          lib/home/surveyresult_page.dart:26:9 •
          prefer_typing_uninitialized_variables
   info • Don't access members with `this` unless avoiding shadowing •
          lib/home/surveyresult_page.dart:27:27 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/home/surveyresult_page.dart:28:45 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/home/surveyresult_page.dart:28:58 • unnecessary_this
   info • Avoid using braces in interpolation when not needed •
          lib/home/surveyresult_page.dart:31:32 •
          unnecessary_brace_in_string_interps
   info • Avoid using braces in interpolation when not needed •
          lib/home/surveyresult_page.dart:32:31 •
          unnecessary_brace_in_string_interps
   info • Don't access members with `this` unless avoiding shadowing •
          lib/home/surveyresult_page.dart:36:7 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/home/surveyresult_page.dart:37:16 • unnecessary_this
   info • Avoid using braces in interpolation when not needed •
          lib/home/surveyresult_page.dart:41:44 •
          unnecessary_brace_in_string_interps
   info • Avoid using braces in interpolation when not needed •
          lib/home/surveyresult_page.dart:42:42 •
          unnecessary_brace_in_string_interps
   info • The value of the local variable 'height' isn't used •
          lib/home/surveyresult_page.dart:54:18 • unused_local_variable
   info • The value of the local variable 'width' isn't used •
          lib/home/surveyresult_page.dart:55:18 • unused_local_variable
   info • Prefer const with constant constructors •
          lib/home/surveyresult_page.dart:59:16 • prefer_const_constructors
   info • Avoid unnecessary containers • lib/home/surveyresult_page.dart:101:13
          • avoid_unnecessary_containers
   info • Avoid `print` calls in production code •
          lib/home/surveyresult_page.dart:106:15 • avoid_print
   info • Prefer typing uninitialized variables and fields •
          lib/home/surveys_page.dart:17:9 •
          prefer_typing_uninitialized_variables
   info • Avoid unnecessary containers • lib/home/surveys_page.dart:31:13 •
          avoid_unnecessary_containers
   info • Avoid `print` calls in production code •
          lib/home/surveys_page.dart:36:15 • avoid_print
   info • Avoid `print` calls in production code •
          lib/home/surveys_page.dart:37:15 • avoid_print
   info • Avoid `print` calls in production code •
          lib/home/surveys_page.dart:38:15 • avoid_print
   info • Unnecessary string interpolation • lib/home/surveys_page.dart:55:35 •
          unnecessary_string_interpolations
   info • Prefer const with constant constructors •
          lib/home/surveys_page.dart:56:30 • prefer_const_constructors
   info • Unused import: 'package:meta/meta.dart' •
          lib/login/bloc/login_bloc.dart:6:8 • unused_import
   info • The operand can't be null, so the condition is always true •
          lib/login/bloc/login_bloc.dart:19:31 • unnecessary_null_comparison
   info • The operand can't be null, so the condition is always true •
          lib/login/bloc/login_bloc.dart:20:35 • unnecessary_null_comparison
   info • Use key in widget constructors • lib/login/login_form.dart:6:7 •
          use_key_in_widget_constructors
   info • 'showSnackBar' is deprecated and shouldn't be used. Use
          ScaffoldMessenger.showSnackBar. This feature was deprecated after
          v1.23.0-14.0.pre. • lib/login/login_form.dart:27:32 •
          deprecated_member_use
   info • Unnecessary string interpolation • lib/login/login_form.dart:28:27 •
          unnecessary_string_interpolations
   info • Avoid unnecessary containers • lib/login/login_form.dart:35:18 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:38:26 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:44:35 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:45:56 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:49:35 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:50:56 • prefer_const_constructors
   info • SizedBox for whitespace • lib/login/login_form.dart:54:21 •
          sized_box_for_whitespace
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:58:34 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:63:34 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:65:36 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:71:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:72:39 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:84:29 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:88:30 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/login/login_form.dart:90:59 • prefer_const_constructors
   info • Prefer declaring const constructors on `@immutable` classes •
          lib/login/login_page.dart:13:3 •
          prefer_const_constructors_in_immutables
   info • The operand can't be null, so the condition is always true •
          lib/login/login_page.dart:14:31 • unnecessary_null_comparison
   info • Prefer const with constant constructors •
          lib/login/login_page.dart:21:16 • prefer_const_constructors
   info • Name source files using `lowercase_with_underscores` •
          lib/main-orig.dart:1:1 • file_names
   info • Use key in widget constructors • lib/main-orig.dart:7:7 •
          use_key_in_widget_constructors
   info • Prefer declaring const constructors on `@immutable` classes •
          lib/main-orig.dart:31:3 • prefer_const_constructors_in_immutables
   info • Prefer const with constant constructors • lib/main-orig.dart:96:13 •
          prefer_const_constructors
   info • Prefer const with constant constructors • lib/main-orig.dart:109:16 •
          prefer_const_constructors
   info • The import of 'dart:typed_data' is unnecessary because all of the used
          elements are also provided by the import of
          'package:flutter/services.dart' • lib/main.dart:2:8 •
          unnecessary_import
   info • Unused import: 'package:myndintranquility/home/categories_page.dart' •
          lib/main.dart:8:8 • unused_import
   info • Avoid using `forEach` with a function literal • lib/main.dart:48:13 •
          avoid_function_literals_in_foreach_calls
   info • Prefer declaring const constructors on `@immutable` classes •
          lib/main.dart:63:3 • prefer_const_constructors_in_immutables
   info • Prefer const with constant constructors • lib/main.dart:71:44 •
          prefer_const_constructors
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/api_model.dart:8:17 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/api_model.dart:9:17 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/category_model.dart:13:15 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/category_model.dart:14:15 • unnecessary_this
   info • Prefer using lowerCamelCase for constant names •
          lib/model/registration_model.dart:2:3 • constant_identifier_names
   info • Prefer using lowerCamelCase for constant names •
          lib/model/registration_model.dart:3:3 • constant_identifier_names
   info • Prefer using lowerCamelCase for constant names •
          lib/model/registration_model.dart:4:3 • constant_identifier_names
   info • Prefer using lowerCamelCase for constant names •
          lib/model/registration_model.dart:5:3 • constant_identifier_names
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:36:17 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:37:14 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:38:12 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:39:17 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:40:19 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:41:18 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/registration_model.dart:42:38 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/resource_model.dart:12:13 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/resource_model.dart:13:14 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/survey_model.dart:21:11 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/survey_model.dart:22:13 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/survey_model.dart:23:14 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/user_model.dart:18:15 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/user_model.dart:19:21 • unnecessary_this
   info • Don't access members with `this` unless avoiding shadowing •
          lib/model/user_model.dart:20:18 • unnecessary_this
   info • Private field could be final • lib/register/register_form.dart:13:8 •
          prefer_final_fields
   info • Private field could be final • lib/register/register_form.dart:14:16 •
          prefer_final_fields
   info • Private field could be final • lib/register/register_form.dart:23:24 •
          prefer_final_fields
   info • Use isEmpty instead of length • lib/register/register_form.dart:33:26
          • prefer_is_empty
   info • Use isEmpty instead of length • lib/register/register_form.dart:40:26
          • prefer_is_empty
   info • Use isEmpty instead of length • lib/register/register_form.dart:49:26
          • prefer_is_empty
   info • Use isEmpty instead of length • lib/register/register_form.dart:57:26
          • prefer_is_empty
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:71:21 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:72:27 • prefer_const_constructors
   info • Avoid unnecessary containers • lib/register/register_form.dart:78:12 •
          avoid_unnecessary_containers
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:82:20 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:88:17 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:185:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:186:52 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:193:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:194:53 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:202:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:203:52 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:211:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:212:49 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:221:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:222:52 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:231:31 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:232:60 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:239:17 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:244:27 • prefer_const_constructors
   info • SizedBox for whitespace • lib/register/register_form.dart:261:17 •
          sized_box_for_whitespace
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:265:30 • prefer_const_constructors
   info • Avoid `print` calls in production code •
          lib/register/register_form.dart:285:35 • avoid_print
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:290:30 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:292:32 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:298:25 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:299:33 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_form.dart:309:39 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_page.dart:11:16 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/register/register_page.dart:13:13 • prefer_const_constructors
   info • Unused import: 'package:meta/meta.dart' •
          lib/repository/user_repository.dart:3:8 • unused_import
   info • Use key in widget constructors • lib/splash/splash_page.dart:3:7 •
          use_key_in_widget_constructors
   info • Prefer const with constant constructors •
          lib/splash/splash_page.dart:6:12 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/splash/splash_page.dart:7:13 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/splash/splash_page.dart:8:16 • prefer_const_constructors
   info • Use key in widget constructors • lib/splash/welcome_page.dart:6:7 •
          use_key_in_widget_constructors
   info • Prefer const with constant constructors •
          lib/splash/welcome_page.dart:27:17 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/splash/welcome_page.dart:28:26 • prefer_const_constructors
   info • Prefer const with constant constructors •
          lib/splash/welcome_page.dart:48:17 • prefer_const_constructors
   info • Unused import: 'package:myndintranquility/main.dart' •
          test/widget_test.dart:11:8 • unused_import
  error • The name 'MyApp' isn't a class • test/widget_test.dart:16:35 •
         creation_with_non_type
```

```
Krishna-Kumars-MacBook-Air:build new$ flutter doctor -v
[✓] Flutter (Channel stable, 2.10.0, on macOS 12.2 21D49 darwin-x64, locale en-IN)
    • Flutter version 2.10.0 at /Users/new/Development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 5f105a6ca7 (6 days ago), 2022-02-01 14:15:42 -0800
    • Engine revision 776efd2034
    • Dart version 2.16.0
    • DevTools version 2.9.2

[✗] Android toolchain - develop for Android devices
    ✗ Unable to locate Android SDK.
      Install Android Studio from: https://developer.android.com/studio/index.html
      On first launch it will assist you in installing the Android SDK components.
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).
      If the Android SDK has been installed to a custom location, please use
      `flutter config --android-sdk` to update to that location.


[✓] Xcode - develop for iOS and macOS (Xcode 13.2)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[!] Android Studio (not installed)
    • Android Studio not found; download from https://developer.android.com/studio/index.html
      (or visit https://flutter.dev/docs/get-started/install/macos#android-setup for detailed instructions).

[✓] Connected device (1 available)
    • Chrome (web) • chrome • web-javascript • Google Chrome 98.0.4758.80

[✓] HTTP Host Availability
    • All required HTTP hosts are available

! Doctor found issues in 2 categories.
```

</details>

Since the other issue asked for it, I'm also attaching my Podfile and pubspec.yaml attached in the zip
[flutter-mac.zip](https://github.com/flutter/flutter/files/8021594/flutter-mac.zip)

**Comment by darshankawar** [comment]
> 6\. Tick the 'For install builds only' in 'Run Scripts' in the Xcode as mentioned [here](https://stackoverflow.com/questions/57919267/command-phasescriptexecution-failed-with-a-nonzero-exit-code-while-trying-to-add)

You probably don't need to tick this option. If ticked, it won't run when you build the app.
Untick that option and run `flutter build ios` and if this throws any error (the one you mentioned in point # 5) provide the entire verbose log of it by running `flutter build ios --verbose` including `ios/podfile`.

**Comment by arunk** [comment]
This is so bizarre. It is working now after I unchecked that option. The only thing I can see is that I might have run the `flutter build ios` command from ssh'ing into the Mac, whereas now I ran the command in the Mac terminal. I tried again to run `flutter build ios` via ssh and it threw an error. But it is working while compiling it from the Mac terminal. So I'll leave this as a note for anyone who faces a similar issue. Thanks for your help @darshankawar
